### PR TITLE
Add Skeleton code for the "add" command

### PR DESCRIPTION
**Pull request by seanieyap**

### CODE REVIEW

File: src/planmysem/common/Utils.java [hunk 1]
```diff
@@ -37,9 +50,129 @@ public static boolean elementsAreUnique(Collection<?> items) {
...
+        }
+
+        return result;
+    }
+
+    /**
+     * Determine recurrence if recess week.
+     */
+    public static boolean containsRecess(Set<String> recurrences) {
+        if (recurrences.contains("recess")) {
```
Comment (marcus-pzj) [inline src/planmysem/common/Utils.java]: return recurrences.contains("recess")

File: src/planmysem/common/Utils.java [hunk 2]
```diff
@@ -37,9 +50,129 @@ public static boolean elementsAreUnique(Collection<?> items) {
...
+        } else {
+            return false;
+        }
+    }
+
+    /**
+     * Determine recurrence if reading week.
+     */
+    public static boolean containsReading(Set<String> recurrences) {
+        if (recurrences.contains("reading")) {
```
Comment (marcus-pzj) [inline src/planmysem/common/Utils.java]: return recurrences.contains("reading")

File: src/planmysem/common/Utils.java [hunk 3]
```diff
@@ -37,9 +50,129 @@ public static boolean elementsAreUnique(Collection<?> items) {
...
+            return true;
+        } else {
+            return false;
+        }
+    }
+    /**
+     * Determine recurrence if normal academic weeks.
+     */
+    public static boolean containsNormal(Set<String> recurrences) {
+        if (recurrences.contains("normal")) {
```
Comment (marcus-pzj) [inline src/planmysem/common/Utils.java]: return recurrences.contains("normal")

**Review by marcus-pzj — CHANGES_REQUESTED** [review]
Some Codacy reviews.

File: src/planmysem/common/Utils.java (hunk 3 above)
Comment (seanieyap) [reply src/planmysem/common/Utils.java]: This and the above has led me to realize that these are unused methods that were supposed to be deleted. Thank you! Changes in next commit.

**Pull request merged**